Add information on how to renew certificate (#54)
bonitasoft/bonita-ici-doc: md/installation_guide.md

Then when using the installer provide this generated file as a keystore, the same password for the keystore and the key, and the same alias.

> #### Renew certificate

Reviewer: ```suggestion
#### Update the certificate
```

> When the certificate expires, it can be renewed by reinstalling the backend (not the storage).

Reviewer: renewing needs to be done on LetsEncrypt (for example), right? It looks like Bonita can renew the certificate.

Author: ok, i'll change that

> When the certificate expires, it can be renewed by reinstalling the backend (not the storage).

Reviewer: ```suggestion
When the certificate expires, renew it on the cerbot of your choice.
To use if on BICI, you need to reinstall the backend (not the storage).
```

> 

Reviewer: ```suggestion
To do so:
```

* stop the backend using `bonita-ici stopApp`

> * start and deploy the backend using either the new certificate or newly generated one using `bonita-ici startApp deploy`

Reviewer: What is the difference between "new certificate" and "newly generated one"?
We can talk maybe. I'm missing something... :/

Author: `generated one` is the one that the installer generate if you choose to do so, and `new certificate` if you provided one

Reviewer: ok. I think it can be clarified, but again, not a big deal.

If the certificate is autogenerated, the Bonita platform JVM needs to be restarted.

> ### Stop the module 

Reviewer: We talk about the stopping the add-on for updating the certificate before we tell Patty how to stop the add-on.
Maybe updating should come after stopping?
Wording: "Stop the Add-on"

Author: yep maybe

Author: in fact its not the same title section +  I prefere  keeping "update the certificate" close to "security configuration"

Reviewer: ok
